### PR TITLE
Add Linear Search

### PR DESCRIPTION
**Pull request by maltejur**
Let's start with a simple one. I am not really familiar with Julia, but I hope this will do.

### CODE REVIEW

File: searches/linear_search.jl [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+            return i
+        end
+    end
+    return -1
+end
+
+array = [1,3,4,7,8,11]
+println(search(array, 3))
+println(search(array, 8))
+println(search(array, 12))
```
Comment (Panquesito7) [inline searches/linear_search.jl:13]: Hmm, probably this should be `11`, as it displays `-1`.
```suggestion
println(search(array, 11))
```

**Review by Panquesito7 — COMMENTED** [review]
👍 Good work.

File: searches/linear_search.jl (hunk 1 above)
Comment (maltejur) [reply searches/linear_search.jl:13]: The point of this line is to show what the function returns if the key is not found.

**Review by Panquesito7 — APPROVED** [review]
Thanks!

**Pull request merged**